Makefile

```diff
@@ -128,6 +128,12 @@ ifeq ($(RUN_SERVER_IN_BACKGROUND),true)
 	RUN_IN_BACKGROUND := &
 endif
 
+ifeq ($(BUILD_ENTERPRISE_READY),true)
```

Comment:
My concern with this is, you are ignoring the user settings when you have enterprise ready builds. I'm ok to fix the current situation,. but I would prefer to iterate over this and find a better way to only run "openldap" container if you are running the tests and you are in a enterprise ready build.

Reply:
Yeah definitely, there's some Makefile magic needed which I don't know enough about. I initially moved the entire condition inside the test-server target, but it wasn't working. Then fell back to the global variable thing.

Some Makefile expertise is most welcome :D